packages/klighd-interactive/src/constraint-classes.ts

```diff
@@ -52,10 +52,11 @@ export class KNode extends RectangularNode implements KGraphElement {
     areChildAreaChildrenRendered = false
     areNonChildAreaChildrenRendered = false
     hasFeature(feature: symbol): boolean {
-        return feature === selectFeature || (feature === moveFeature && (this.parent as KNode).properties.interactiveLayout)
+        return feature === selectFeature || (feature === moveFeature && (this.parent as KNode).properties.get("org.eclipse.elk.interactiveLayout") as boolean)
```

Comment:
We try to use single quotes for strings in this project. This is, however, not enforced (yet).